### PR TITLE
Fixed a bug in sample_posterior_predictive()

### PR DESCRIPTION
**Pull request by digicosmos86**
The bug originates from the handling of `size` parameter passed to `rng_fn`

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def data_angle():
 def data_ddm_reg():
     # Generate some fake simulation data
     intercept = 0.3
-    x = np.random.uniform(0.5, 0.2, size=1000)
+    x = np.random.uniform(0.5, 0.7, size=1000)
     y = np.random.uniform(0.4, 0.1, size=1000)
```
Comment (AlexanderFengler) [inline tests/conftest.py:38]: intercept = 1.5
x, y = np.random.uniform(-.5, .5, size = 1000)

File: tests/test_utils.py [hunk 2]
```diff
@@ -64,11 +64,13 @@ def test_get_alias_dict():
 
     assert alias_regression["c(rt, response)"] == "rt,response"
     assert alias_regression["Intercept"] == "v_Intercept"
+    assert alias_regression["x"] == "v_x"
```
Comment (AlexanderFengler) [inline tests/test_utils.py:67]: is this test_util still ok after demoting `v` from default parent?

**Review by AlexanderFengler — COMMENTED** [review]
Looks good just minimal comment!

File: tests/test_utils.py (hunk 2 above)
Comment (digicosmos86) [reply tests/test_utils.py:67]: Yep. In this case the parent is still `"v"`. With `alias_regression_a` the parent becomes `"a"` which is why I had to change the tests

File: tests/conftest.py (hunk 1 above)
Comment (digicosmos86) [reply tests/conftest.py:38]: fixed

**Review by AlexanderFengler — APPROVED** [review]
ok looks good.

**Pull request merged**